lib/lightning_web/live/tokens_live/index.html.heex

```diff
@@ -106,30 +105,29 @@
         </.tr>
       <% end %>
       <%= for token <- @tokens do %>
-        <.tr id={"token-#{token["id"]}"}>
+        <.tr id={"token-#{token.id}"}>
           <.td>
             <div class="flex rounded overflow-auto p-2 bg-blue-100 w-auto justify-between">
               <code class="my-auto whitespace-nowrap py-2 px-4 align-baseline text-base font-medium leading-none">
                 <span>
-                  <%= token["token"] %>
+                  <%= mask_token(token) %>
```

Comment:
@mtuchi it's ok to call this function in a template because the `for` should be able to invalidate extra renders..